starter-dev-e2e/cypress/e2e/dashboard.cy.ts

```diff
@@ -48,3 +49,36 @@ describe("When there is proper dashboard page responses", () => {
       );
   });
 });
+
+describe("When there is proper empty dashboard page responses", () => {
+  beforeEach(() => {
+    cy.interceptGraphQLCalls(View.Dashboard, InterceptResponse.Empty);
+    cy.interceptRestCalls(View.Dashboard, InterceptResponse.Empty);
+  });
+
+  // Error displayed instead of empty list.
+  // Angular-Apollo has empty message but displays error; Next-React empty & error msg displays same
```

Comment:
Yeah this seems to be a bug in both -- let's write the tests for the desired behavior and let it be filed as a bug against the showcases. I assume all we need to do is uncomment below?

Reply:
Yes, I used the empty gists text from the angular-apollo project.